src/ParseServer.js

```diff
@@ -243,7 +243,6 @@ class ParseServer {
     Config.validate(AppCache.get(appId));
     this.config = AppCache.get(appId);
     Config.setupPasswordValidator(this.config.passwordPolicy);
-    hooksController.load();
```

Comment:
And you validation logic should throw if there's an error loading the hooks, inside the hooks controller.